lambdas/sqs-message-remover/src/index.js

```diff
@@ -4,17 +4,40 @@ const get = require('lodash/get');
 
 const { sqs } = require('@cumulus/aws-client/services');
 const { deleteSQSMessage } = require('@cumulus/aws-client/SQS');
+const { deleteS3Object } = require('@cumulus/aws-client/S3');
 const {
   getSfEventMessageObject,
   getSfEventStatus,
   isFailedSfStatus,
   isSfExecutionEvent,
   isTerminalSfStatus,
 } = require('@cumulus/common/cloudwatch-event');
+const { envUtils } = require('@cumulus/common');
 const Logger = require('@cumulus/logger');
 
 const logger = new Logger({ sender: '@cumulus/sqs-message-remover' });
 
+/**
+ * Deletes archived SQS Message from S3
+ *
+ * @param {Object} messageId - SQS message ID
+ * @returns {void}
+ */
+async function deleteArchivedMessage(messageId) {
+  const bucket = envUtils.getRequiredEnvVar('system_bucket', process.env);
+  const stackName = envUtils.getRequiredEnvVar('stackName', process.env);
+  const key = `${stackName}/archived-incoming-messages/${messageId}`;
+  if (bucket && key) {
```

Comment:
I think the `if` statement here is unnecessary if we're already doing a try/catch (which is good). The try/catch will handle logging the error if either bucket/key is missing. And actually, with the `if` statement, it's perhaps less clear what happened because no error gets logged

Reply:
[`5223038` (#2132)](https://github.com/nasa/cumulus/pull/2132/commits/522303829adf7b5c63fec589dced18ebcf084cf4)